pinot-common/src/main/java/com/linkedin/pinot/common/response/broker/BrokerResponseNative.java

```diff
@@ -229,6 +233,26 @@ public void setNumGroupsLimitReached(boolean numGroupsLimitReached) {
     _numGroupsLimitReached = numGroupsLimitReached;
   }
 
+  @JsonProperty("numBytesReadInFilter")
```

Comment:
This may not really be needed. I believe JSON library we use can handle the '_' prefix.